File: app/src/main/java/com/example/user/biblejournal/model/database/verse/VerseDao.java
package com.example.user.biblejournal.model.database.verse;

import androidx.room.Dao;
import androidx.room.Query;

@Dao
public interface VerseDao {
    @Query("SELECT book, chapter, verse, content from bible WHERE book=:book AND chapter=:chapter AND verse=:verse")
    VerseEntity getVerse(int book, int chapter, int verse);
}

File: app/src/main/java/com/example/user/biblejournal/model/database/verse/VerseDao.kt
package com.example.user.biblejournal.model.database.verse

import androidx.room.Dao
import androidx.room.Query

@Dao
interface VerseDao {
    @Query("SELECT book, chapter, verse, content from bible WHERE book=:book AND chapter=:chapter AND verse=:verse")
    fun getVerse(book: Int, chapter: Int, verse: Int): VerseEntity
}
